pkgs/misc/cups/drivers/cups-drv-rastertosag-gdi/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, fetchzip
+, fetchpatch
+, cups
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "rastertosag-gdi";
+  version = "0.1";
+  src = fetchzip {
+    url = "https://www.openprinting.org/download/printing/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "1ldplpv497j8vhw24sksg3fiw8c5pqr0wajajh7p5xpvb6zlcmvw";
+  };
+  patches = [
+    # port to python 3
+    ( fetchpatch {
+      url = "https://sources.debian.org/data/main/r/${pname}/0.1-7/debian/patches/0001-${pname}-python3.patch";
+      sha256 = "1l3xbrs67025595k9ba5794q3s74anizpbxwsshcfhmbrzd9h8hg";
+    })
+  ];
+  format = "other";
+  nativeBuildInputs = [ (lib.attrsets.getBin cups) ];
```

Comment:
```suggestion
  nativeBuildInputs = [ (lib.getBin cups) ];
```

Reply:
Thanks for the hint and commit.  That's fine with me.  I usually use subsets of `lib` as I feel these are more readable and more self-documenting.  But it doesn't really make a difference here.